lms/static/sass/shared/_header.scss

```diff
@@ -349,18 +358,22 @@ header.global-new {
   box-shadow: 0 1px 5px 0 $shadow-l1;
   background: $header-bg;
 
-  nav {
+  .wrapper-header {
     @include clearfix();
     @include box-sizing(border-box);
-    width: grid-width(12);
     height: 74px;
     margin: 0 auto;
     padding: 17px 0;
+    min-width: 880px;
+
+    &.rwd {
+      min-width: 0px;
+    }
```

Comment:
Can you updated so min-width is only being set for not .rwd
&:not(.rwd) {
    min-width: 880px;
}